### ISSUE TITLE
Potential risk of loss of lists when user suspended/removed from service?

### ISSUE THREAD

**Issue by davekasten** [bluesky-social/proposals#10]
As I understand https://github.com/bluesky-social/proposals/tree/main/0001-user-lists-replygating-and-thread-moderation so far, it seems that lists are owned by users.  

That makes sense. 

Does that mean that if a user gets kicked from a server, that list goes away permanently (or at least until they join a new server, if one exists)?  That's how I think I read the proposal's implications, but if that's not correct, the below is irrelevant; I'm writing the below scenario on the assumption I'm correct.

Potential scenario for impact:  Since it _seems_ that lists are becoming a greater part of the self-help trust and safety features already on bluesky than previous sites, there's a potential risk here at scale that's worth considering. 

Imagine Alice runs a list that is very helpful for a community, for whatever reason.  Alice has posted for a long time, and eventually draws attention of bad people, who screen through all of her posts to ID reasons for her to be suspended from the service she's hosted on.  

Ok, maybe in the future decentralized world, she can move to another server.  But 1. that isn't now and 2. even if so, what if it happens while Alice is on vacation and away from her phone for a week?  The community loses that list for an indefinite period of time.  That seems like it could have a lot of blast radius at scale, particularly if those lists are being used to protect a community from harassment. 

Do you want to allow static copies of lists to persist (at least by default) for a period of time after a user is suspended?

**Comment by jphastings** [comment]
I was thinking over this scenario too; lists seem incredibly powerful, particularly in the community managed cases, but being permanently attached to a single user could produce continued inescapable obligation.

If Alice created a list that’s used extensively by a community, but later decides that she needs a break from this community, she must either take a break from that ATProto account, or (assuming lists can be transferred) permanently give up ownership.

A similar situation occurs if Alice decides to move to a PDS with rules that the list contravenes.

(All the above assumes that lists are attached to users, and do not become a first-class entity of ATProto; apologies if that wasn’t an intended part of the 0001 draft!)

**Comment by pfrazee** [comment]
This is all correct and it's a tricky balance. You almost want an alt exclusively dedicated to hosting the list, which could then be managed by multiple people.
